src/lucky_migrator/create_table_statement.cr

```diff
@@ -113,6 +114,28 @@ class LuckyMigrator::CreateTableStatement
     raise "index on #{@table_name}.#{column} already exists"
   end
 
+  # Adds a references column and index given a model class and references option.
+  macro belongs_to(model_class, references = nil)
+    {% optional = model_class.is_a?(Generic) %}
+
+    {% if optional %}
+      {% class_underscore = model_class.type_vars.first.stringify.underscore %}
+    {% else %}
+      {% class_underscore = model_class.names.first.stringify.underscore %}
+    {% end %}
```

Comment:
And then you could do `{% underscored_class = model_class.stringify.gsub(/\?/, "").underscore %}` without the if/else